source/blog/2011-06-18-doctrine-2-1-rc1.md

```diff
@@ -19,7 +19,7 @@ should give you lots of motivation to try out 2.1 with your projects.
 We integrated a pretty substantial Annotation Reader refactoring just
 today. Please test if this still works with your code-base using
 annotations. See the [UPGRADE\_TO\_2\_1
-file](https://github.com/doctrine/doctrine2/blob/master/UPGRADE_TO_2_1)
+file](https://github.com/doctrine/orm/blob/master/UPGRADE_TO_2_1)
```

Comment:
`https://github.com/doctrine/orm/blob/master/UPGRADE.md#upgrade-to-21`

Please also put the syntax of the markdown link in one line.

The 2.1 branch doesn't highlight markdown formats because of a missing file suffix, which is why I use the master-Branch here.

Reply:
> Please also put the syntax of the markdown link in one line.

I don't get what do you mean

Reply:
I mean it was done like that because of 120 chars per line rule I suppose

Reply:
Nevermind that "one line" comment. It already got fixed elsewhere. :-)